charts/private-action-runner/templates/_helpers.tpl

```diff
@@ -95,13 +104,12 @@
       },
```

Comment:
Known issue: the comma on this line will become a trailing comma if `$c.applicationName` is defined and `$c.searchPath` is undefined, causing the JSON output to be broken. Intentionally not fixing as it will require messy logic here and `connectionCredentials` is deprecated